Index: src/routes/index.tsx
===================================================================
--- src/routes/index.tsx
+++ src/routes/index.tsx
@@ -3,8 +3,9 @@
 import {
   PageHeader,
   PageHeaderDescription,
   PageHeaderTitle,
+  Widget,
 } from '@prisma-crud/ui-shared';
 import { createFileRoute } from '@tanstack/react-router';
 
 export const Route = createFileRoute('/')({
@@ -19,7 +20,8 @@
         <PageHeaderDescription>
           This is the home page of a generated app.
         </PageHeaderDescription>
       </PageHeader>
+      <Widget label="Imported from ui-shared" />
     </div>
   );
 }
