Index: src/emails/index.ts
===================================================================
--- src/emails/index.ts
+++ src/emails/index.ts
@@ -1,3 +1,7 @@
+export { default as AccountVerificationEmail } from './auth/account-verification.email.js';
+
+export { default as PasswordChangedEmail } from './auth/password-changed.email.js';
+export { default as PasswordResetEmail } from './auth/password-reset.email.js';
 /* TPL_EMAIL_TEMPLATES:START */
 export { default as TestEmail } from './test.email.js';
 /* TPL_EMAIL_TEMPLATES:END */
