Index: prisma/schema.prisma
===================================================================
--- prisma/schema.prisma
+++ prisma/schema.prisma
@@ -45,14 +45,13 @@
   id                 String        @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
   filename           String
   mimeType           String        @map("mime_type")
   encoding           String?
-  size               Int
+  size               Int?
   category           String
   adapter            String
   storagePath        String        @map("storage_path")
-  referencedAt       DateTime?     @map("referenced_at") @db.Timestamptz(3)
-  expiredAt          DateTime?     @map("expired_at") @db.Timestamptz(3)
+  pendingUpload      Boolean       @default(false) @map("pending_upload")
   uploaderId         String?       @map("uploader_id") @db.Uuid
   updatedAt          DateTime      @default(now()) @updatedAt @map("updated_at") @db.Timestamptz(3)
   createdAt          DateTime      @default(now()) @map("created_at") @db.Timestamptz(3)
   todoListCoverPhoto TodoList[]
