Index: src/prisma/seed.ts
===================================================================
--- src/prisma/seed.ts
+++ src/prisma/seed.ts
@@ -4,9 +4,20 @@
 
 async function main(): Promise<void> {
   try {
     // any additional seed logic goes here
-    await prisma.$queryRaw`SELECT 1;`;
+    await prisma.blogPost.createMany({
+      data: [
+        {
+          title: 'First post',
+          content: 'Hello world!',
+        },
+        {
+          title: 'Second post',
+          content: 'Goodbye world!',
+        },
+      ],
+    });
   } finally {
     await prisma.$disconnect();
   }
 }
